Police-Data-Accessibility-Project/prod-to-dev-migration
# Mc 417 data sources crud

### Fixes

* Supports https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/429

### Description

* Add permissions for data sources
* Add logic which deletes from `column_permission` and `relation_column` tables before running main body of `upload_relation_configurations_to_db.py`
* Add relation_column_permission_view to database
* Update test table to utilize for automated tests
* Update `agency_source_link` so that foreign keys receive cascading updates from parent tables.

### Testing

* Run branch in migration script in `automation.pdap.io`, confirm it runs successfully
* Confirm schema and data updates are as expected in database.

### Performance

* Performance impact marginal.

### Docs

* Not applicable.

### Breaking Changes

* Not applicable.
